apps/spruce/src/components/Content/Layout.tsx

```diff
@@ -55,7 +56,8 @@ export const Layout: React.FC = () => {
         />
       )}
       <FloatingContent>
-        <TaskStatusIconLegend />
+        {/* TODO: Use user's beta feature setting instead. */}
```

Comment:
Let's include the ticket number here
```suggestion
        {/* TODO DEVPROD-10203: Use user's beta feature setting instead. */}
```